src/dict.c

```diff
@@ -758,6 +761,12 @@ void dictRelease(dict *d)
     if (dictIsRehashing(d) && d->type->rehashingCompleted)
         d->type->rehashingCompleted(d);
 
+    if (d->type->sizeChanged) {
+        /* Subtract the size of all buckets. */
+        unsigned long bucket_count = DICTHT_SIZE(d->ht_size_exp[0]) + DICTHT_SIZE(d->ht_size_exp[1]);
+        d->type->sizeChanged(d, -(long long)bucket_count);
```

Comment:
we have `dictBuckets`, maybe just use `d->type->sizeChanged(d, -dictBuckets(d));`

Reply:
done with [`bbbc52b` (#13984)](https://github.com/redis/redis/pull/13984/commits/bbbc52b784fce3e164e4d18b1c76a27eab1b5e3d)